rollbar-java/src/main/java/com/rollbar/notifier/sender/result/ResultCode.java

```diff
@@ -1,4 +1,4 @@
-package com.rollbar.notifier.sender;
+package com.rollbar.notifier.sender.result;
 
 /**
  * Represents expected response codes from POSTing an item to Rollbar.
```

Comment:
Do we really need a class for this? I would imagine there's already something in the standard library for mapping the response codes to errors/messages.

Reply:
As the SyncSender and JsonSerializer this comes from the previous version. We could use instead of it just int numbers mapped by the constants in the [HttpURLConnection](http://docs.oracle.com/javase/7/docs/api/java/net/HttpURLConnection.html). However some codes are missing, like the 429 mapped to `Too many requests`. At the same time this could work as a quick view/doc of the response codes our API delivers.